packages/grafana-ui/src/components/Button/FullWidthButtonContainer.tsx

```diff
@@ -1,13 +1,13 @@
 import { css, cx } from '@emotion/css';
-import React, { FC } from 'react';
+import React from 'react';
 
 import { stylesFactory } from '../../themes';
 
 export interface Props {
   className?: string;
 }
 
-export const FullWidthButtonContainer: FC<Props> = ({ className, children }) => {
+export const FullWidthButtonContainer = ({ className, children }: React.PropsWithChildren<Props>) => {
```

Comment:
According to the https://github.com/grafana/grafana/blob/main/contribute/style-guides/frontend.md#how-to-declare-functional-components we should use function instead of const.

We could do something like that:

```
export function FullWidthButtonContainer ({ className, children }:React.PropsWithChildren<Props>){
  const styles = getStyles();

  return <div className={cx(styles, className)}>{children}</div>;
};
```